examples/storage.py

```diff
@@ -14,13 +21,21 @@ class MyApplication(asab.Application):
 	async def main(self):
 		storage = self.get_service("asab.StorageService")
 
-		u = storage.upsertor("test-collection")
+		u = storage.upsertor("test-collection", 1)
```

Comment:
dtto as above
And I'm wondering why you have to specify `objid` now ... it wasn't a requirement before.

Reply:
You had change arguments of upsertor method from
```
def upsertor(self, collection, origObj=None):
```
to
```
def upsertor(self, collection, obj_id, version=None):
```
not passing `obj_id` throws `TypeError: upsertor() missing 1 required positional argument: 'obj_id'`

Should we default `obj_id` to `None` ?

Reply:
Yes, `obj_id = None` seems to be viable.